packages/base/Breadcrumbs/src/BreadcrumbsItem/BreadcrumbsItem.tsx

```diff
@@ -45,12 +37,15 @@ export const BreadcrumbsItem: OverridableComponent<Props> = forwardRef<
     ...rest
   } = props
   const Component = active ? Active : as || 'span'
-  const classes = useStyles()
 
   const titleCase = useTitleCase(propsTitleCase)
 
   return (
-    <Component ref={ref} className={cx(classes.root, className)} {...rest}>
+    <Component
+      ref={ref}
+      className={twMerge('text-[14px] font-semibold', className)}
```

Comment:
The original style is actually bold. I see that we don't have `bold` on [picasso-tailwind](https://github.com/toptal/picasso/blob/8e87b967e7700b9a1109f0b9db5463d0e62281e7/packages/picasso-tailwind/src/index.js#L57), I believe we should add it with the same value as [700](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#bold)
```suggestion
      className={twMerge('text-[14px] font-bold', className)}
```

Reply:
@augustobmoura visual tests passed, and if we don't have bold, it means we don't have it in BASE. Seems like it was a bug before 🤔

Reply:
Visual tests aren't failing because they don't test boldness. Both Storybook stories override the element of the component to `Link`, by using the property `as` ([1](https://github.com/toptal/picasso/blob/ebbfab5cda838fb342004556bf3d81366b09dacf/packages/base/Breadcrumbs/src/Breadcrumbs/story/Default.example.tsx#L39), [2](https://github.com/toptal/picasso/blob/ebbfab5cda838fb342004556bf3d81366b09dacf/packages/base/Breadcrumbs/src/Breadcrumbs/story/SimpleLinks.example.tsx#L15)). The `Link` components override font-weight and this change is not made visible.

While this is fine for tests on Picasso, we can't assure the result will be the same on product portals and alike. If `bold` is not a BASE token, we should use [arbitrary values](https://tailwindcss.com/docs/font-weight#arbitrary-values) for this case.

```suggestion
      className={twMerge('text-[14px] font-[700]', className)}
```

I would also ask for maybe an example or Cypress test that doesn't override the `Breadcrumb.Item` component with the `as` prop, as it might affect visual tests